packages/select/src/select.vue

```diff
@@ -64,7 +64,7 @@
         v-model="query"
         @input="debouncedQueryChange"
         v-if="filterable"
-        :style="{ width: inputLength + 'px', 'max-width': inputWidth - 42 + 'px' }"
+        :style="{ 'flex-grow': '1', width: inputLength / 208 + '%', 'max-width': inputWidth - 42 + 'px' }"
```

Comment:
If I don't modify this line, I find it also works fine. By the way, what does 208 mean?

Reply:
208 is the width of el-select__tags (I know it should be dynamic and I am pushing new commit to make it dynamic)
& What do you mean by If I don't modify this line. (Not working with updated line?)